File: tools/copyrighter/copyright_param.rb
class CopyrightParam

  attr_accessor :name, :value

  def initialize(name, value)
    @name = name
    @value = value
  end

  def decorated_name
    "${#{@name}}"
  end

  def to_s
    "{#{@name}:#{@value}}"
  end

end

File: tools/copyrighter/copyright_param.py
class CopyrightParam:
    def __init__(self, name, value):
        self.name = name
        self.value = value

    @property
    def decorated_name(self):
        return f"${{{self.name}}}"

    def __str__(self):
        return f"{{{self.name}:{self.value}}}"
